netci.groovy

```diff
@@ -89,6 +89,13 @@ def static setMachineAffinity(def job, def os, def architecture) {
 }
 
 def static isGCStressRelatedTesting(def scenario) {
+    // The 'gcstress15_pri1r2r' scenario is a basic scenario.
+    // Detect it and make it a GCStress related.
+    if (scenario == 'gcstress15_pri1r2r')
```

Comment:
It would probably be safe to match any pattern containing 'gcstress', but this is probably fine.  It's not like it already isn't incredibly inconsistent.